packages/webapp/src/components/CustomSignUp/index.js

```diff
@@ -0,0 +1,61 @@
+import styles from './styles.scss';
+import Logo from '../../assets/images/signUp/logo.svg';
+import LineBreak from '../../assets/images/signUp/lineBreak.svg';
+import Button from '../Form/Button';
+import Input from '../Form/Input';
+import React from 'react';
+import Footer from '../Footer';
+import GoogleLoginButton from '../../containers/GoogleLoginButton';
+
+export default function PureCustomSignUp({ inputs = [{}], ssoSignUp, onSubmit, disabled }) {
+  var color = '';
+  disabled ? (color = '#D4DAE3') : (color = '#FCE38D');
+
+  return (
+    <>
+      <div className={styles.home}>
+        <div className={styles.lander}>
+          <div className={styles.greetContainer}>
+            <img src={Logo} />
+            <div className={styles.ssoButton}>
+              <GoogleLoginButton />
+            </div>
+            <div className={styles.lineBreak}>
+              <img src={LineBreak} />
+            </div>
+
+            <div className={styles.continueButton}>
+              <Input
+                style={{ width: 312 }}
+                className={styles.manualInputContainer}
+                {...inputs[0]}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <Footer
+        style={{ position: 'sticky', bottom: '0' }}
```

Comment:
Might be better to put styling in `styles.scss` and use `className`

Reply:
Done